staging/src/k8s.io/client-go/examples/third-party-resources/client.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/rest"
+)
+
+func NewClient(cfg *rest.Config) (*rest.RESTClient, *runtime.Scheme, error) {
+	groupVersion := schema.GroupVersion{
+		Group:   ExampleResourceGroup,
+		Version: ExampleResourceVersion,
+	}
+
+	schemeBuilder := runtime.NewSchemeBuilder(func(scheme *runtime.Scheme) error {
+		scheme.AddKnownTypes(
+			groupVersion,
+			&Example{},
+			&ExampleList{},
+			&metav1.ListOptions{},
+			&metav1.DeleteOptions{},
+			&metav1.Status{},
+		)
+		return nil
+	})
+
+	scheme := runtime.NewScheme()
+	if err := schemeBuilder.AddToScheme(scheme); err != nil {
+		return nil, nil, err
+	}
+	if err := api.AddToScheme(scheme); err != nil {
```

Comment:
This is a suggested workaround from there: https://github.com/kubernetes/kubernetes/issues/26003#issuecomment-285674832

Without this line, if you keep application with watcher running (in this example the application shuts down quickly, so you won't notice it, but if you just add a sleep for 10 minutes in the end, you'll see it), there is an error in the log occuring every 5 minutes:
```
2017/03/15 00:24:54 [WATCH] OnAdd /apis/tpr.k8s.io/v1/namespaces/default/examples/example1
E0315 00:29:54.550267   11041 streamwatcher.go:109] Unable to decode an event from the watch stream: unable to decode watch event: no kind "Status" is registered for version "v1"
E0315 00:29:54.553813   11041 streamwatcher.go:109] Unable to decode an event from the watch stream: unable to decode watch event: no kind "Status" is registered for version "v1"
2017/03/15 00:29:55 [WATCH] OnUpdate oldObj: /apis/tpr.k8s.io/v1/namespaces/default/examples/example1
2017/03/15 00:29:55 [WATCH] OnUpdate newObj: /apis/tpr.k8s.io/v1/namespaces/default/examples/example1
```